src/P0wnedPassword/Request/Client.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+/*
+ * This file is part of the RollerworksPasswordStrengthValidator package.
+ *
+ * (c) Sebastiaan Stok <s.stok@rollerscapes.net>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Rollerworks\Component\PasswordStrength\P0wnedPassword\Request;
+
+use GuzzleHttp\Psr7\Request;
+use Http\Client\Exception\HttpException;
+use Http\Client\Exception as HttpException2;
+use Http\Client\HttpClient;
+use Psr\Log\LoggerInterface;
+
+class Client
```

Comment:
Should be marked as `@internal` to prevent accidentally using this outside of this package.

Reply:
what's wrong with using it outside the package? Suppose someone wants to check the password but not fail as it would through a validator?

Reply:
Oh wait we can't make it internal because then you can't "use" it for service injection 😅 
OK no problem then.